src/software/simulated_tests/simulated_er_force_sim_test_fixture.cpp

```diff
@@ -25,56 +29,97 @@ void SimulatedErForceSimTestFixture::SetUp()
 {
     LoggerSingleton::initializeLogger(TbotsGtestMain::logging_dir);
 
-    mutable_thunderbots_config->getMutableAiControlConfig()->getMutableRunAi()->setValue(
-        !TbotsGtestMain::stop_ai_on_start);
+    friendly_mutable_thunderbots_config->getMutableAiControlConfig()
+        ->getMutableRunAi()
+        ->setValue(!TbotsGtestMain::stop_ai_on_start);
 
     // The simulated test abstracts and maintains the invariant that the friendly team
     // is always the yellow team
-    mutable_thunderbots_config->getMutableSensorFusionConfig()
+    friendly_mutable_thunderbots_config->getMutableSensorFusionConfig()
         ->getMutableOverrideGameControllerDefendingSide()
         ->setValue(true);
-    mutable_thunderbots_config->getMutableSensorFusionConfig()
+    friendly_mutable_thunderbots_config->getMutableSensorFusionConfig()
         ->getMutableDefendingPositiveSide()
         ->setValue(false);
 
     // Experimentally determined restitution value
-    mutable_thunderbots_config->getMutableSimulatorConfig()
+    friendly_mutable_thunderbots_config->getMutableSimulatorConfig()
         ->getMutableBallRestitution()
         ->setValue(0.8);
     // Measured these values from fig. 9 on page 8 of
     // https://ssl.robocup.org/wp-content/uploads/2020/03/2020_ETDP_ZJUNlict.pdf
-    mutable_thunderbots_config->getMutableSimulatorConfig()
+    friendly_mutable_thunderbots_config->getMutableSimulatorConfig()
         ->getMutableSlidingFrictionAcceleration()
         ->setValue(6.9);
-    mutable_thunderbots_config->getMutableSimulatorConfig()
+    friendly_mutable_thunderbots_config->getMutableSimulatorConfig()
         ->getMutableRollingFrictionAcceleration()
         ->setValue(0.5);
 
     // The simulated test abstracts and maintains the invariant that the friendly team
     // is always defending the "negative" side of the field. This is so that the
     // coordinates given when setting up tests is from the perspective of the friendly
     // team
-    mutable_thunderbots_config->getMutableSensorFusionConfig()
+    enemy_mutable_thunderbots_config->getMutableSensorFusionConfig()
         ->getMutableFriendlyColorYellow()
         ->setValue(true);
```

Comment:
is the enemy supposed to be yellow?

Reply:
I think so

Reply:
Yes because from the enemy AI perspective it is "yellow" and the the friendly AI is "blue" - thats why we flip the coords for blue.